packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -258,6 +258,37 @@ describe("PanelResizeHandle", () => {
     });
   });
 
+
+  describe("event handlers when rendered in a separate document", () => {
+    let separateWindowDocument: Document;
+    beforeEach(() => {
+      // @ts-expect-error
+      global.IS_REACT_ACT_ENVIRONMENT = true;
+
+      separateWindowDocument = document.implementation.createHTMLDocument();
+      container = separateWindowDocument.createElement("div");
+      separateWindowDocument.body.appendChild(container);
+      jest.spyOn(separateWindowDocument.body,"addEventListener");
+      jest.spyOn(document.body,"addEventListener");
+      expectedWarnings = [];
+      root = createRoot(container);
+    });
+
+    it("should add the pointerup event to the separate document NOT the main document", () => {
+      const { leftElement } = setupMockedGroup();
+
+      act(() => {
+        dispatchPointerEvent("pointerdown", leftElement);
+        dispatchPointerEvent("pointerup", leftElement);
+      });
+
+      expect(separateWindowDocument.body.addEventListener).toHaveBeenCalledWith("pointerup", expect.anything(), expect.anything());
```

Comment:
This checks if addEventListener is called to add the handlePointerUp to the body element. I am looking into a stricter matcher than expect.anything